docs/how_we_work/deploy_flow.md

```diff
@@ -51,25 +51,25 @@ Since development is a space for experimental changes, don't count on the develo
 
 For initial development and testing, we are currently using a sandbox on cloud.gov: https://cloud.gov/pricing/.
 
-### Continuous deployment to dev 
+### Continuous deployment to dev
 
 Unless we create many dev apps and write functionality similar to [Heroku Review Apps](https://devcenter.heroku.com/articles/github-integration-review-apps) (one app per in-progress PR), we will only have one dev app for an engineering team of three. How should devs trigger deploys to the dev app?
 
-Our plan is to allow any dev to trigger a deploy to the dev app via a [git tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging). (See ["Executing workflows for a git tag" in the Circle docs](https://circleci.com/docs/2.0/workflows/#executing-workflows-for-a-git-tag).) This will give devs the flexibility to deploy to dev and test out new code at any time, without waiting for a review process or a git merge process. 
+Our plan is to allow any dev to trigger a deploy to the dev app via a merge to the `development` branch.
```

Comment:
[question] I think it runs even before it gets merged, it will run in a PR if it attempts to merge right?  We may want to express that too.

Reply:
No, I don't think so. We're only deploying to the development environment if it gets merged into development, otherwise we would overwrite development with each new PR to the development branch and I don't think we want that, right? @amymok

Reply:
@amymok I made a junk change with a junk PR #237  just to verify the above is true, and you can see the deploy is skipped if the branch is not `development`. LMK if this is not what you expected.

<img width="1077" alt="Screen Shot 2021-12-22 at 10 08 36 AM" src="https://user-images.githubusercontent.com/10871517/147129762-73f506c2-530d-49cf-862a-7c83b08c3b9d.png">

Reply:
Oh good good.  Thanks for verifying.  Yes, we only want to deploy when it is merged.  But tests will run regardless.